src/app/components/viewer/viewer.component.ts

```diff
@@ -205,7 +205,7 @@ export class AppViewerComponent implements OnInit, OnDestroy {
           this.previousNodeId = nearest.left;
           this.nextNodeId = nearest.right;
           this.nodeId = this.node.id;
-          this.fileName = this.node.name;
+          this.fileName = this.node.name + this.node?.properties?.['cm:versionLabel'];
```

Comment:
this does not look correct, with the name being "my-file.txt" and "versonLabel" is "1", it will show "my-file.txt1" ?

Reply:
Nope @DenysVuika  https://github.com/Alfresco/alfresco-ng2-components/blob/e2b88ddab0e58776544a073effda8dcb2e3c74d4/lib/core/viewer/components/viewer.component.ts#L392 seems we don't actually use it for now is a workaround

Reply:
this one works as a temp solution, and as per discussions, we can refactor ADF viewer for the next version